### PR TITLE
Includes restructure

### PR DESCRIPTION
**Pull request by guihomework**
This PR changes the header file location to easily permits installing the files and allow external plugins to re-use classes or plugin displays of rviz_tactile_plugins

### CODE REVIEW

**Comment by guihomework** [comment]
Seems I broke the runtime now... 
```
[ERROR] [1661794645.104098490]: PluginlibFactory: The plugin for class 'rviz_tactile_plugins/Tactile State Display' failed to load. 
 Error: Failed to load library catkin_ws_noetic/devel/lib//librviz_tactile_plugins.so. 
   Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML.
 Error string: Could not load library (Poco exception = catkin_ws_noetic/devel/lib//librviz_tactile_plugins.so: undefined symbol: _ZNK4rviz7tactile17TactileVisualBase10metaObjectEv)
```
somebody solved this type of issues when adding the header files in the build https://answers.ros.org/question/267548/rviz-throws-undefined-symbol-exception-when-including-from-include-folder/?answer=268163

will try this now

**Comment by guihomework** [comment]
the solution I copied from the forum works, additionally to the missed export.
if you don't like it, please suggest a cleaner way

File: rviz_tactile_plugins/include/rviz_tactile_plugins/tactile_array_visual.h [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 #pragma once
 
-#include "tactile_visual_base.h"
+#include "rviz_tactile_plugins/tactile_visual_base.h"
```
Comment (rhaschke) [inline rviz_tactile_plugins/include/rviz_tactile_plugins/tactile_array_visual.h]: Please change double quotes `"..."` to angle brackets `<...>` to indicate that the [header is to be found in the search path and not locally](https://stackoverflow.com/questions/3162030/difference-between-angle-bracket-and-double-quotes-while-including-heade#:~:text=When%20you%20use%20angle%20brackets,search%20the%20include%20path%20list.).

**Review by rhaschke — COMMENTED** [review]
LGTM. Just a minor nitpick...

**Review by rhaschke — APPROVED** [review]

**Comment by rhaschke** [comment]
Thanks!

**Pull request merged**